.github/workflows/main.yml

```diff
@@ -4,25 +4,17 @@ on: [push, pull_request]
 
 jobs:
   flatpak:
-
     runs-on: ubuntu-latest
-
     container:
-      image: elementary/docker:unstable
+      image: docker.io/bilelmoussaoui/flatpak-github-actions
```

Comment:
Note that you could use a different image here. It's an image that has fdo 20.08 sdk & platform pre-installed as they are the most common ones.